ProcessMaker/Http/Middleware/VerifyActiveSession.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace ProcessMaker\Http\Middleware;
+
+use Closure;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Cache;
+use ProcessMaker\Models\UserSession;
+use Symfony\Component\HttpFoundation\Response;
+use Illuminate\Support\Facades\Crypt;
+
+class VerifyActiveSession
+{
+    /**
+     * Handle an incoming request.
+     *
+     * @param  \Closure(\Illuminate\Http\Request): (\Symfony\Component\HttpFoundation\Response)  $next
+     */
+    public function handle(Request $request, Closure $next): Response
+    {
+        if (!$request->hasHeader('Authorization')) {
+            $user = \Auth::user();
+            $activeSession = Cache::get('user_' . $user->id . '_active_session');
```

Comment:
What happens if the user logins in different browsers or devices?
Do the last one will overwrite the previous ones?

Reply:
Yes, the Cache will be updated to Active and with a new timestamp.

Reply:
So a user that starts a session in browser, its session could be closed by timeout of other session started in a different device